hauler_all_the_things.sh

```diff
@@ -30,21 +30,32 @@ function info_ok { echo -e "$GREEN" "ok" "$NO_COLOR" ; }
 
 export PATH=$PATH:/usr/local/bin
 
+# Package Manager 
+if [[ -f /etc/debian_version ]]; then 
+  export PKG_MANAGER='apt-get';
```

Comment:
just for the build of the package. Because I am running on WSL ubuntu. This choice is also made because github actions are by default ubuntu. So it help me. but the deployment goes on RHEL8.